components/user_group_popover/user_group_popover.tsx

```diff
@@ -0,0 +1,229 @@
+    };
+
+    return (
+        <Popover
+            {...props}
+            id='user-group-popover'
+        >
+            <Body role='complementary'>
+                <Header>
+                    <Heading>
+                        <Title
+                            className='overflow--ellipsis text-nowrap'
+                        >
+                            {group.display_name}
+                        </Title>
+                        {canManageGroup ? (
+                            <SimpleTooltip
+                                id={`user-group-popover-${group.id}`}
+                                content='Edit group'
+                            >
+                                <EditButton
+                                    className='user-group-popover_edit-button btn-icon'
+                                    aria-label='Edit Group'
+                                    onClick={openViewGroupModal}
+                                >
+                                    <PencilOutlineIcon/>
+                                </EditButton>
+                            </SimpleTooltip>
+                        ) : null}
+                    </Heading>
+                    <Subtitle>
+                        <span className='overflow--ellipsis text-nowrap'>{'@'}{group.name}</span>
+                        <Dot>{' • '}</Dot>
+                        <FormattedMessage
+                            id='user_group_popover.memberCount'
+                            defaultMessage='{member_count} {member_count, plural, one {Member} other {Members}}'
+                            values={{
+                                member_count: group.member_count,
+                            }}
+                            tagName={NoShrink}
+                        />
+                    </Subtitle>
+                    {group.member_count > 10 ? (
+                        <SearchBar
+                            type='text'
+                            className='user-group-popover_search-bar'
+                            placeholder={Utils.localizeMessage('user_group_popover.searchGroupMembers', 'Search members')}
+                            inputPrefix={<i className={'icon icon-magnify'}/>}
+                            value={searchTerm}
+                            onChange={handleSearch}
+                            useLegend={false}
+                        />
+                    ) : null}
+                </Header>
+                <GroupMemberList
+                    group={group}
+                    hide={hide}
+                    isSearchLoading={isSearchLoading}
+                    showUserOverlay={showUserOverlay}
+                />
+            </Body>
+        </Popover>);
+};
+
+const Body = styled.div`
+    width: 264px;
+`;
+
+const Header = styled.div`
+    padding: 16px 20px;
+`;
+
+const Title = styled.span`
+    flex: 1 1 auto;
+`;
+
+const EditButton = styled.button`
+    width: 24px;
+    height: 24px;
```

Comment:
I think the hover effect is a little small and needs to be 28px, that would match it with the figma